parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -3257,6 +3257,462 @@ arryDMLSubClause
     : LP_ typeName (COMMA_ varrayType)? RP_
     ;
 
+createMaterializedView
+    : CREATE MATERIALIZED VIEW materializedViewName
+      (OF typeName )?
+        ( LP_ (scopedTableRefConstraint | mvColumnAlias) (COMMA_ (scopedTableRefConstraint | mvColumnAlias))* RP_ )?
+        ( ON PREBUILT TABLE ( (WITH | WITHOUT) REDUCED PRECISION)?
+        | physicalProperties?  (CACHE | NOCACHE)? parallelClause? buildClause?
+        )
+        ( USING INDEX ( (physicalAttributesClause | TABLESPACE tablespaceName)+ )*
+        | USING NO INDEX
+        )?
+        createMvRefresh?
+        (FOR UPDATE)?
+        ( (DISABLE | ENABLE) QUERY REWRITE )?
+        AS selectSubquery
+    ;
+
+rowLimitingClause
```

Comment:
Hi @zhangfengcdt , can you take a look at DMLStatement? I think rowLimitingClause and selectSubquery already exist in DMLStatement.

Reply:
@strongduanmu I was trying to import the "DMLStatement" in the DDL g4 statement, however, this causes a circular import since DML also imports DDL g4 file.

I think the circular dependency could be removed by moving some of the definitions into the base g4 file, but that seems to be out of scope for this change. 

Could you advise how to proceed? Thanks!

Reply:
Which DDL statements will DML statements depend on? This seems unreasonable.